build/docker/zap-baseline.py

```diff
@@ -314,7 +314,8 @@ def main(argv):
                 time.sleep(5)
                 logging.debug('Delay passive scan check ' + str(delay - (datetime.now() - start_scan).seconds) + ' seconds')
 
-        zap_wait_for_passive_scan(zap)
+        # Allow twice as long as the time taken to spider for the passive scanning
+        zap_wait_for_passive_scan(zap, mins * 60 * 2)
```

Comment:
Should it be specifiable by the commandline (in case someone determines it is too short for their use)?